sdk/src/main/java/com/oursky/authgear/latte/WebView.kt

```diff
@@ -8,14 +8,18 @@ import android.view.inputmethod.InputMethodManager
 import com.oursky.authgear.AuthgearException
 
 @SuppressLint("SetJavaScriptEnabled")
-internal class WebView(context: Context) : android.webkit.WebView(context) {
+internal class WebView(context: Context, inspectable: Boolean = false) : android.webkit.WebView(context) {
```

Comment:
`webContentsDebuggingEnabled: Boolean`

No default; follow the naming convention.